extensions.bzl

```
@@ -0,0 +1,71 @@
+load("//:rules.bzl", bb_macro = "buildbuddy")
+
+def _ext_impl(mctx):
+    found_root_modules = False
+    root_modules = struct()
```

```suggestion
    found_root_module = False
    root_module = struct()
```

and below as there is only a single one.

---

Good call. Fixed.